sdk/core/azure-core-tracing-opentelemetry/README.md

```diff
@@ -0,0 +1,73 @@
+
+
+# Azure Core Tracing OpenTelemetry client library for Python
+
+## Getting started
+
+Install the opentelemetry python for Python with [pip](https://pypi.org/project/pip/):
+
+```bash
+pip install azure-core-tracing-opentelemetry --pre
+```
+
+Now you can use opentelemetry for Python as usual with any SDKs that is compatible
+with azure-core tracing. This includes (not exhaustive list), azure-storage-blob, azure-keyvault-secrets, azure-eventhub, etc.
+
+## Key concepts
+
+* You don't need to pass any context, SDK will get it for you
+
+## Examples
+
+There is no explicit context to pass, you just create your usual opentelemetry tracer and
+call any SDK code that is compatible with azure-core tracing. This is an example
+using Azure Monitor exporter, but you can use any exporter (Zipkin, etc.).
+
+```python
+from opentelemetry.ext.azure_monitor import AzureMonitorSpanExporter
+
+from opentelemetry import trace
+from opentelemetry.sdk.trace import Tracer
+
+from azure.core.settings import settings
+from azure.core.tracing.ext.opentelemetry_span import OpenTelemetrySpan
+
+from azure.storage.blob import BlobServiceClient
+
+# Declare that you want to trace Azure SDK with OpenTelemetry
+settings.tracing_implementation = OpenTelemetrySpan
```

Comment:
It would be super neat if we somehow could just assign a string here. But that can wait.

I would add some comments about what is "new"/unique for someone who doesn't know how to use the opentelemetry library. E.g. importing settings and specifying which implementation to use. Somehow, the example as is makes it look way more complicated than it is...

Reply:
Doing a string implies pre-declaration in azure-core. I thought we didn't want that?

I re-orged the Readme, let me know what you think

Reply:
I was thinking more of a string to figure out what package to load type of scheme...

Reply:
This readme reads better - great!